debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -273,7 +272,7 @@ private void migrateTable(final Queue<ChangeTable> schemaChangeCheckpoints)
             }
             if (schema.tableFor(currentTable.getSourceTableId()) == null) {
```

Comment:
@jpechane, I'm wondering about this: wouldn't we miss to record the schema for an `ALTER`ed table?